client/src/pages/CreateOrganizationProfile.js

```diff
@@ -327,6 +312,19 @@ const CreateOrgProfile = (props) => {
               <Link to="/terms-conditions">Terms and Conditions</Link>
             </StyledCheckbox>
```

Comment:
You have to add the rule to this to be required as per the issue. This two checkboxes has to be checked before the user can create a profile.